BUILDING-RELEASES.adoc

```diff
@@ -0,0 +1,54 @@
+== How To Build Releases Using Maven Release Plugin and Nexus Repositories
+
+The following steps will tell you what you need to do in order to build a release for public consumption.
+
+Before you can do anything, you must ensure your Maven environment and project are configured properly.
+
+* Put your Maven public repository credentials in your ~/.m2/settings.xml file. For example, the Hawkular parent pom defines the public maven repository that we use for releases under the name "jboss-releases-repository", so in your settings.xml you will need something like this:
+[source,xml]
+<servers>
+  <server>
+    <id>jboss-releases-repository</id>
+    <username>your.maven.repository.username</username>
+    <password>your.maven.repository.password</password>
+  </server>
+</servers>
+
+* Make sure your Maven project's main pom.xml defines an <scm> section with at least a <developerConnection> entry. Here's an example:
```

Comment:
I have also not seen this problem. e.g. I only have <scm> in the bus parent, all the child modules do not have it. bus released ok.